app/src/main/java/co/kr/bemyplan/ui/purchase/after/AfterPurchaseActivity.kt

```diff
@@ -31,4 +34,12 @@ class AfterPurchaseActivity : AppCompatActivity() {
         val mapView = MapView(this)
         binding.mapView.addView(mapView)
     }
+
+    private fun initFragment() {
+        val fragmentManager = supportFragmentManager
+        val fragmentTransaction = fragmentManager.beginTransaction()
+        val fragment = DailyContentsFragment()
+        fragmentTransaction.add(R.id.fcv_daily_context, fragment)
+        fragmentTransaction.commit()
```

Comment:
이건 정말 간단한 건데 사실 이렇게 두 줄로 따로 안적어두 되구

```suggestion
        fragmentTransaction.add(R.id.fcv_daily_context, fragment).commit()
```
이렇게만 작성해도 괜찮아요~!!

Reply:
wow 낭비했네요 감사합니다